index.d.ts

```diff
@@ -14,6 +14,7 @@ declare namespace WebpackPwaManifest {
     type Direction = 'ltr' | 'rtl' | 'auto';
     type Display = 'fullscreen' | 'standalone' | 'minimal-ui' | 'browser';
     type Orientation = 'any' | 'natural' | 'landscape' | 'landscape-primary' | 'landscape-secondary' | 'portrait' | 'portrait-primary' | 'portrait-secondary';
+    type CrossOrigin = 'use-credentials' | 'anonymous';
```

Comment:
should this default to `'anonymous'` or `undefined`?

Reply:
The default value should be undefined. So this is covered by making the crossorigin option optional right?